### DISCUSSION TITLE
Visual indication of copy-mode

### DISCUSSION THREAD

**Discussion by diego898** [catppuccin/tmux#420] [Help]
Hello! I'd like to have some kind of visual indication of entering copy-mode - perhaps the status indicator that changes color when you press the prefix, can enter a "peach" state?

**Comment by diego898** [comment]
Solved this with:

```
# If in COPY mode, set to peach color, different icon, and text.
set -ogq "@catppuccin_session_icon" "#{?pane_in_mode,📋, }"
set -ogq "@catppuccin_session_color" "#{?pane_in_mode,#{E:@thm_peach},#{?client_prefix,#{E:@thm_red},#{E:@thm_green}}}"
set -ogq "@catppuccin_session_text" "#{?pane_in_mode,#[bold] COPY #[default], #S}"
```

Still tweaking - will leav this open for a few days in case anyone else wants to chime in before I close

**Comment by diego898** [comment]
Just saw someone had the same idea and similar fix in share your config thread

**Comment by kjnsn** [comment]
I got something working with: `set -g status-left "#{?pane_in_mode,#[bg=#{@thm_peach} fg=#{@thm_crust}] COPY 📋 ,}"`
